app/charts/combo/combo-state.ts

```diff
@@ -187,7 +187,7 @@ export const useDualAxisMargins = ({
   return getMargins({
     left,
     right,
-    bottom: bottom - SINGLE_LINE_AXIS_LABEL_HEIGHT / 2,
+    bottom,
```

Comment:
@noahonyejese not sure if it's the reason, but combo chart X labels are cut, see e.g. here:

<img width="905" alt="Screenshot 2025-02-28 at 09 42 18" src="https://github.com/user-attachments/assets/6c43ad8c-2588-4b7d-a9b0-9b453020ca74" />